src/main/java/com/hazelcast/web/WebFilterConfig.java

```diff
@@ -53,6 +53,7 @@
     private static final String DEFERRED_WRITE = "deferred-write";
     private static final String USE_REQUEST_PARAMETER = "use-request-parameter";
     private static final String TRANSIENT_ATTRIBUTES = "transient-attributes";
+    private static final String NOTIFY_CLUSTER = "notify-cluster";
```

Comment:
I think the same goes for this parameter, it should be a bit more explanatory.

Reply:
Renamed as `keep-remote-active`